lib/explorer/data_frame.ex

```diff
@@ -129,17 +129,65 @@ defmodule Explorer.DataFrame do
 
   @doc """
   Reads a IPC file into a dataframe.
+
+    ## Options
+
+      * `with_columns` - List with name of columns to be selected.
```

Comment:
Is `with_columns` the name we typically use for picking columns in Explorer?

Reply:
We normally use "`select`", do you think "`with_selection`" or simply "`select`" is a better choice?

Reply:
I agree with @josevalim :+1:

Reply:
Nice!

Made those changes on https://github.com/elixir-nx/explorer/commit/26d05270590af6ef8ba500616a254ef330427acb and https://github.com/elixir-nx/explorer/pull/152/commits/e62e61cf1654fa0eff9e31be479d7bf96964707f